### DISCUSSION TITLE
Feature suggestion: Auto close TLDR tooltips after X seconds.

### DISCUSSION THREAD

**Discussion by nobewayo** [Lailloken/Exile-UI#473] [Ideas & Feedback]
I don't need to see it for more than a second or two, to see what I need to press.
Anything green = click. Anything red = don't. 

A custom (optional) timer to automaticly close it would be really great.

Sometimes I struggle to close it because ESC either isn't responding or there's suddenly mobs around me and I stress myself out trying to click on it, which also doesn't seem to be responsive all the time.

**Comment by Lailloken** [comment]
This feature needs to toggle the visibility of ground loot, which happens when it's activated and hidden. The tool can't send the second key-press based on a timer because that's against ToS.

**Comment by nobewayo** [comment]
An argument could be made that showing/hiding loot isn't a server action, but only client side. But I get your point, it's better to stay on the safe side. I'll probably just implement it myself then haha.
It works fine without hiding the loot though, I changed the hotkey for it away from Z, without changing the loot toggle hotkey in PoE.
